CONTRIBUTING.md

```diff
@@ -32,6 +32,26 @@ channel on Freenode](http://webchat.freenode.net/?channels=grpc), or send an
 email to [grpc-io@googlegroups.com](grpc-io@googlegroups.com) to discuss
 beforehand.
 
+## Pull Requests & Commits
+
+We have few conventions for keeping history clean and making code reviews easier
+for reviewers:
+
+* First line of commit messages should be in format of
+
+  `package-name: summary of change`
+
+  for example:
+
+  `core: fix infinite loop`
```

Comment:
Thanks for doing this.  If you wouldn't mind, could you change the template on line 42 to be

```
package-name: summary of change
Where the summary finishes the sentence: "This commit improves gRPC to ____________.".
```

This format was roughly copied from Go and the Linux kernel.  Preferably, the reason starts with a lower case letter, though it seems this has not been consistently followed.  I personally don't care which, as long as it's one of them.  (it makes it way easier to scan when building the release notes!)

Also, if you wouldn't mind, could you additionally add a slightly more complicated example.  Something like:

```
core,netty,interop-testing:  add capacitive duractance to turbo encabulators
````